pkg/kube/translation/apisix_pluginconfig.go

```diff
@@ -0,0 +1,36 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package translation
+
+import (
+	configv2beta3 "github.com/apache/apisix-ingress-controller/pkg/kube/apisix/apis/config/v2beta3"
+	apisixv1 "github.com/apache/apisix-ingress-controller/pkg/types/apisix/v1"
+)
+
+func (t *translator) TranslateApisixPluginConfig(apc *configv2beta3.ApisixPluginConfig) (*apisixv1.PluginConfig, error) {
+	pluginMap := make(apisixv1.Plugins)
+	if len(apc.Spec.Plugins) > 0 {
+		for _, plugin := range apc.Spec.Plugins {
+			for k, v := range plugin {
+				// Here, it will override same key.
```

Comment:
I think either the override behavior should be documented or we should enrich the CRD schema so that the overridden won't happen.

Reply:
The CRD schema currently does not have this verification capability. we can document the behavior.